test/jasmine/tests/plot_api_test.js

```diff
@@ -1,20 +1,24 @@
 var Plotly = require('@src/plotly');
+var Plots = require('@src/plots/plots');
+var Scatter = require('@src/traces/scatter');
+var Bar = require('@src/traces/bar');
+var Legend = require('@src/components/legend');
 
 describe('Test graph_obj', function() {
     'use strict';
 
     describe('Plotly.restyle', function() {
         beforeEach(function() {
-            spyOn(Plotly.Plots, 'previousPromises');
             spyOn(Plotly, 'plot');
-            spyOn(Plotly.Scatter, 'arraysToCalcdata');
-            spyOn(Plotly.Bar, 'arraysToCalcdata');
-            spyOn(Plotly.Plots, 'style');
-            spyOn(Plotly.Legend, 'draw');
+            spyOn(Plots, 'previousPromises');
+            spyOn(Scatter, 'arraysToCalcdata');
+            spyOn(Bar, 'arraysToCalcdata');
+            spyOn(Plots, 'style');
+            spyOn(Legend, 'draw');
```

Comment:
Interesting. 

These methods are being `spyOn`'ed correctly? Is it become `src/plots/plots.js` still requires `src/plotly.js` ?